src/Build/Logging/BinaryLogger/ProjectImportsCollector.cs

```diff
@@ -131,6 +131,14 @@ private void AddFileCore(string filePath)
             }
 
             var fileInfo = new FileInfo(filePath);
+
+#if FEATURE_SYMLINK_TARGET
+            if (fileInfo.Length == 0 && fileInfo.Exists && !string.IsNullOrEmpty(fileInfo.LinkTarget))
```

Comment:
Why is this core only?

Reply:
`LinkTarget` is not available in Full FW - so supporting this functionality in full FW would require involved, special-cased code. I'd do that once this is asked by at least single user, that needs in on full FW.